app/views/pages/api_client_timeout.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l"><%= t("pages.api_client_timeout.title") %></h1>
+    <p class="govuk-body">
+      <%= t("pages.api_client_timeout.body_text") %>
+    </p>
+
+    <p class="govuk-body">
+      <%= t("pages.api_client_timeout.email_help",
```

Comment:
The other error pages don't use the locales, but maybe they should because we use locales a lot generally.

Should we add a ticket to switch over the other error pages?

Reply:
Yeah I think a tech debt ticket to get everything translated would be good